src/components/Docs/SnippetViewer/CheckRequestViewer.tsx

```diff
@@ -126,6 +126,36 @@ var response = await fgaClient.Check(new CheckRequest(new TupleKey() {
       });
 
 // response.Allowed = ${allowed}`;
+    case SupportedLanguage.PYTHON_SDK:
+      return `
+# Run a check
+body = openfga_sdk.model.check_request.CheckRequest(
+    tuple_key=openfga_sdk.model.tuple_key.TupleKey(
+        user="${user}",
+        relation="${relation}",
+        object="${object}",
+    ),
+${
+  contextualTuples
+    ? `
+    contextual_tuples=openfga_sdk.model.contextual_tuple_keys.ContextualTupleKeys(
+        tuple_keys=[
+            ${contextualTuples
+              .map(
+                (tuple) =>
+                  `openfga_sdk.model.tuple_key.TupleKey(user="${tuple.user}", relation="${tuple.relation}", object="${tuple.object}")`,
```

Comment:
This seems a bit verbose. Can we show without the full import path (and maybe the imports at the top)?

Reply:
the problem is that the setup (where the import is defined) is used by different call (such as check, read, expand, etc.).  The alternative is to have the setup section import everything (but this makes the setup section very long)

Reply:
Or we leave a comment in the setup section to tell them to import other models as needed

Reply:
Yeah something like this at the top would help, just to reduce verbosity:
```
# from openfga_sdk.model.tuple_key import TupleKey
```